packages/mds-geography-author/tests/api.spec.ts

```diff
@@ -25,6 +25,7 @@
 
 import { ApiServer } from '@mds-core/mds-api-server'
 import db from '@mds-core/mds-db'
```

Comment:
Could we get rid of `db` entirely?

Reply:
There's a whole 'nother ticket for that.